### PR TITLE
added syntheticwsi recipe

### PR DESCRIPTION
**Pull request by ThomasWollmann**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/syntheticwsi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{% set SYNTHETICWSI_VERSION = "0.91" %}
+
+package:
+  name: syntheticwsi
+  version: {{ SYNTHETICWSI_VERSION }}
+
+source:
+  fn: v{{ SYNTHETICWSI_VERSION }}.zip
+  url: https://github.com/apgrgr/SyntheticWSI/archive/v{{ SYNTHETICWSI_VERSION }}.zip
+  sha1: 442dd8c8b58c2fd20bf7c50e2f5e62a3ed09cce3
```
Comment (bgruening) [inline recipes/syntheticwsi/meta.yaml]: Mh, this is not a jar file isn't it?

Comment (ThomasWollmann) [inline recipes/syntheticwsi/meta.yaml]: not yet

**Comment by bgruening** [comment]
Is there no way to ask for a jar file.

**Comment by ThomasWollmann** [comment]
Do you mean from the original authors?

**Comment by bgruening** [comment]
Yes.

**Comment by ThomasWollmann** [comment]
I can give it a try. However don't invest too much in this package it's not as important as https://github.com/bioconda/bioconda-recipes/pull/3438.

**Comment by johanneskoester** [comment]
We have migrated our build system from Travis CI to Circle CI. If you would like to continue working on this PR, please update with latest upstream master and reopen this PR.

**Pull request closed without merging**